src/authorizations/components/redesigned/CustomApiTokenOverlay.tsx

```diff
@@ -9,13 +9,19 @@ import {
   AlignItems,
   FlexDirection,
   ComponentSize,
+  Button,
+  ComponentColor,
+  ButtonShape,
 } from '@influxdata/clockface'
 
+import ResourceAccordion from './ResourceAccordion'
+
 interface OwnProps {
   onClose: () => void
 }
 
 export const CustomApiTokenOverlay: FC<OwnProps> = props => {
+  const resources = ['telegrafs', 'buckets']
```

Comment:
FYI: Resources hardcoded right now but will change in a different issue, where we will use the api to get these resources.